### PR TITLE
(#862) Deployment to upgrade all Software

### PR DESCRIPTION
**Pull request by st3phhays**
## Description Of Changes
This adds documentation on how to upgrade all Software with an automatic Deployment Plan.

## Motivation and Context
This reflects changes made in Chocolatey Central Management 0.12.0

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made
* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue
* #862

### CODE REVIEW

File: input/en-us/central-management/usage/website/software.md [hunk 1]
```diff
@@ -21,6 +21,22 @@ The Software main page lists all installed software in any computers that have c
...
+
+From the Software main page in Chocolatey Central Management, click the **Create New Deployment Plan** button and select the **Upgrade All Software** option.
+
+![Button to upgrade all Software from the Software main page](/assets/images/software/ccm-software-details-upgrade-all-software.png)
+
+Clicking this option will create a New Deployment Plan. This Deployment Plan will create one Deployment Step per Software. 
+
+![Automatically created Deployment Plan showing a Deployment Step for each piece of Software](/assets/images/software/ccm-software-upgrade-all-steps.png)
+
+Each Deployment Step will a Temporary Group that contains all the Computers the Software is currently installed on.
```
Comment (AdmiringWorm) [inline input/en-us/central-management/usage/website/software.md:34]:
```suggestion
Each Deployment Step will create a Temporary Group that contains all the Computers the Software is currently installed on.
```

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
The referenced images are not included, until they are, this PR cannot be approved.

File: input/en-us/central-management/usage/website/software.md (hunk 1 above)
Comment (st3phhays) [reply input/en-us/central-management/usage/website/software.md]: This has been updated.

**Comment by st3phhays** [comment]
> The referenced images are not included, until they are, this PR cannot be approved.

I have tracked the images that need updated.

**Review by vexx32 — APPROVED** [review]
Apart from the images that will need to be added later, this looks good :3

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Review by corbob — DISMISSED** [review]

File: input/en-us/central-management/usage/website/software.md [hunk 2]
```diff
@@ -21,6 +21,22 @@ The Software main page lists all installed software in any computers that have c
 
 ![Software main page](/assets/images/software/ccm-software-main.png)
 
+## Upgrade All Software
+
+From the Software main page in Chocolatey Central Management, click the **Create New Deployment Plan** button and select the **Upgrade Outdated Software** option.
+
+![Button to upgrade all outdated Software from the Software main page](/assets/images/software/ccm-software-details-upgrade-all-software.png)
```
Comment (gep13) [inline input/en-us/central-management/usage/website/software.md:28]:
```suggestion
![Button to upgrade all outdated Software from the Software main page](/assets/images/software/ccm-software-upgrade-all-software.png)
```

**Review by gep13 — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/usage/website/software.md (hunk 2 above)
Comment (st3phhays) [reply input/en-us/central-management/usage/website/software.md]: This has been fixed.

File: input/en-us/central-management/usage/website/software.md [hunk 3]
```diff
@@ -21,6 +21,22 @@ The Software main page lists all installed software in any computers that have c
 
 ![Software main page](/assets/images/software/ccm-software-main.png)
 
+## Upgrade All Software
+
+From the Software main page in Chocolatey Central Management, click the **Create New Deployment Plan** button and select the **Upgrade Outdated Software** option.
```
Comment (gep13) [inline input/en-us/central-management/usage/website/software.md:26]:
```suggestion
From the Software main page, click the **Create New Deployment Plan** button and select the **Upgrade Outdated Software** option.
```

**Review by gep13 — CHANGES_REQUESTED** [review]

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Review by gep13 — APPROVED** [review]
LGTM!

**Pull request merged**